app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -88,6 +91,40 @@ class ProjectInformationActivity : AppCompatActivity() {
         }
     }
 
+    private fun setApplyButtonText(applyButton: Button, applied: Boolean?) {
+        if (applied == null)
+            applyButton.text = LOADING_STRING
+        else if (applied)
+            applyButton.text = UNAPPLY_STRING
+        else
+            applyButton.text = APPLY_STRING
```

Comment:
It seems that there is a little code dupplicate (applyButton.text = ...), maybe this will be shorter : 
```suggestion
        applyButton.text = when (applied) {
            null -> LOADING_STRING
            true -> UNAPPLY_STRING
            false -> APPLY_STRING
        }
```